### PR TITLE
CVE-2016-5143 and CVE-2016-1651

### PR DESCRIPTION
**Pull request by kalebdavis**

### CODE REVIEW

File: cves/CVE-2016-1651.yml [hunk 1]
```diff
@@ -15,33 +15,46 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
```
Comment (David-Egan) [inline cves/CVE-2016-1651.yml]: Could you specify what aspect of the vulnerability allows for memory to be read past the end of allocated space.

File: cves/CVE-2016-1651.yml [hunk 2]
```diff
@@ -15,33 +15,46 @@ description_instructions: |
...
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - :commit: 83dfeae5a988dbd17f11b409f32ef577be02c152
-  :note: ''
-vccs: []
-upvotes_instructions: |
+  :note: |
+    This is the commit that updates the version of PDFium, but the actual VCCs
```
Comment (David-Egan) [inline cves/CVE-2016-1651.yml]: How does the new version of PDFium fix the vulnerability

File: cves/CVE-2016-1651.yml [hunk 3]
```diff
@@ -94,10 +109,16 @@ interesting_commits:
...
     emerging themes?
   commits:
-  - commit: 
-    note: 
-  - commit: 
-    note: 
+  - commit: 6cf012af4954807255ce7cdb5b92a20f74d34e6d, bff247d92c315f9aab2edb4aeaf0ef0b74ba321f
+    note: |
+      These commits seem to touch on the future vulnerability, but they're plugging
+      leaks rather than fixing the entire problem. Specifically, these two commits deal
```
Comment (David-Egan) [inline cves/CVE-2016-1651.yml]: Wording for "Specifically, these two commits deal with edge cases where..." seems a little awkward. Consider revising it.

File: cves/CVE-2016-5143.yml [hunk 4]
```diff
@@ -26,22 +29,28 @@ reviews:
...
 - :commit: 554517a4587bfb0071bcd3c7eff6645a0b06d72a
   :note: ''
-vccs: []
+vccs:
+- commit: 36b91b446fbc09f334d6c7c8575477330ea9d39a
+  note: This commit was specifically to remove any `../` found in the sourceURL, meaning they were trying to fix another vulnerability and accidentally introduced this one
+- commit: e8ecfb59d4f906e0ab40b6046406b8af1366cb10
+  note: This was a commit that attempted to validate the remote base passed in, but did it in an incomplete way.
+- commit: c5eecf67fd8d5e8d24d2d4d9489753d2c8cf6c59
+  note: This commit was intended to sanitize the remote front end URL for any old front ends. It sanitized it, but in a way that left holes.
```
Comment (David-Egan) [inline cves/CVE-2016-5143.yml]: It's not completely clear what you mean by the sanitation leaving "holes"

**Comment by David-Egan** [comment]
CVE-2016-5143: 4 upvotes
CVE-2016-161: 2 upvotes

File: cves/CVE-2016-1651.yml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 CVE: CVE-2016-1651
-CWE: 
+CWE:
```
Comment (samshiffman321) [inline cves/CVE-2016-1651.yml]: what is the CWE number this vulnerability best corresponds to?

File: cves/CVE-2016-5143.yml [hunk 6]
```diff
@@ -15,7 +15,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
```
Comment (samshiffman321) [inline cves/CVE-2016-5143.yml]: this description could use a bit more detail. aim for 1-2 full paragraphs.

File: cves/CVE-2016-5143.yml [hunk 7]
```diff
@@ -15,7 +15,10 @@ description_instructions: |
...
+  This vulnerability affects the DevTools subsystem of Google Chrome. At some point,
+  a hostname, remoteBase parameter, and remoteFrontendUrl parameter are passed
```
Comment (kayladavis) [inline cves/CVE-2016-5143.yml]: Try to explain this without function names and maybe explain it slightly more in depth if possible.

File: cves/CVE-2016-1651.yml [hunk 8]
```diff
@@ -15,33 +15,48 @@ description_instructions: |
...
+    beginning of the application.
+upvo  tes_instructions: |
   For the first round, ignore this upvotes number.
 
   For the second round of reviewing, you will be giving a certain amount of
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes: 
+upvotes: 2
```
Comment (kayladavis) [inline cves/CVE-2016-1651.yml]: 👍 upvote

**Pull request merged**